### PR TITLE
chore: update release query options [SPA-507]

### PR DESCRIPTION
**Pull request by ChidinmaOrajiaku**
## Summary

Add `order` to release query options.

## Checklist (check all before merging)

- [ ] Both unit and integration tests are passing
- [ ] There are no breaking changes
- [ ] Changes are reflected in the documentation

When adding a new method:

- [ ] The new method is exported through the default and plain CMA client
- [ ] All new public types are exported from `./lib/export-types.ts`
- [ ] Added a unit test for the new method
- [ ] Added an integration test for the new method
- [ ] The new method is added to the documentation

### CODE REVIEW

**Comment by luizfonseca** [comment]
can we also add the `sys.createdBy.sys.id` filter?

**Comment by ChidinmaOrajiaku** [comment]
> can we also add the `sys.createdBy.sys.id` filter?

Done

**Review by Spring3 — APPROVED** [review]

File: lib/entities/release.ts [hunk 1]
```diff
@@ -35,6 +35,15 @@ export interface ReleaseQueryOptions {
...
   limit?: number
+  /**
+   * Order releases by
+   * @default -sys.updatedAt
+   * */
+  order?: string
+  /**
+   * Filters by creator of release
+   */
+  'sys.createdBy.sys.id'?: string
```
Comment (luizfonseca) [inline lib/entities/release.ts:46]: I think it's 
```suggestion
  'sys.createdBy.sys.id[in]'?: string
```

**Review by luizfonseca — APPROVED** [review]
Minor fix but 🚀

**Pull request merged**